src/components/experiment-tracking/parallel-coordinates/parallel-coordinates.js

```diff
@@ -76,11 +83,81 @@ export const ParallelCoordinates = ({ metricsData, selectedRuns }) => {
   };
 
   const handleMouseOverMetric = (e, key) => {
+    const runsCount = graph.find((each) => each[0] === key)[1].length;
     setHoveredAxisG(key);
+
+    const rect = e.target.getBoundingClientRect();
+
+    let x, direction;
+
+    if (window.innerWidth - rect.x > tooltipMaxWidth) {
+      x = e.clientX - sideBarWidth;
+      direction = 'right';
+    } else {
+      x = e.clientX - sideBarWidth - sideBarWidth / 2;
+      direction = 'left';
+    }
+    const y = rect.y - 140;
+
+    setShowTooltip({
+      content: {
+        label1: 'Metrics Name',
+        value1: key,
+        label2: 'Runs Count',
+        value2: runsCount,
+      },
+      direction,
+      pos: { x, y },
+      visible: true,
+    });
   };
 
   const handleMouseOutMetric = () => {
     setHoveredAxisG(null);
+    setShowTooltip({
+      pos: { x: -500, y: -500 },
+      visible: false,
+    });
+  };
```

Comment:
Could you do this instead? `setShowTooltip(tooltipDefaultProps)`?

Reply:
ah yeah good point 👍 thats better thank you